hydrus/resources.py

```diff
@@ -143,21 +141,17 @@ def post(self, id_: str, path: str) -> Response:
                             type_=object_["@type"],
                             session=get_session(),
                             api_name=get_api_name())
-                        headers_ = [
-                            {"Location": "{}/{}/{}".format(
+                        headers_ = [{"Location": "{}{}/{}/{}".format(
```

Comment:
This is wrong. You can't have four placeholders and give only three strings in format. It will raise an IndexError exception.
Why is this being done anyway?

Reply:
string formatting is the proper way to do this if we wanto to support Python3.5+.

@de-sh i am starting to dubt that you run tests properly, as these things are quite evident

Reply:
@Mec-iS,

Reply:
Still don't see what the extra placeholder is for. I am surprised that Python does not give an error for this. Are you running tests on the correct branch, i.e. the one you made the changes to or on the original branch?

Reply:
@chrizandr  It looks fine to me
```
headers_ = [{"Location": "{}{}/{}/{}".format(
get_hydrus_server_url(), get_api_name(), path, object_id)}]
```
There are 4 placeholders and 4 strings

Reply:
Looks like earlier we had 3 placeholders and 4 values (` get_hydrus_server_url(), get_api_name(), path, object_id`). Doesn't seem right. Anyways, the PR looks good to me.